dev.Dockerfile

```diff
@@ -15,12 +15,12 @@ ENV CGO_ENABLED=0
 RUN make install
 
 # FINAL IMAGE
-FROM alpine as final
+FROM bookworm as final
```

Comment:
Same here, we want to keep the slim alpine base image, as the Debian is much larger.